### ISSUE TITLE
$100 bounty for supporting multiple units

### ISSUE THREAD

**Issue by outdooricon** [sdague/waterfurnace#9]
I would really, really like to have support for my two separate units to work in Home Assistant. I'd be willing to pay $100 through venmo or "buy me a coffee" to the engineer who builds the solution. Would anyone be willing to take it on? 

I know @heffneil was asking for this too at https://github.com/home-assistant/core/issues/45556

Cc @sdague @mdallaire

**Comment by heffneil** [comment]
We are using ccutlers gem on a pi with our units that makes water furnace talk mqtt. Private message him or me and we will continue the conversation. It works really well.

**Comment by outdooricon** [comment]
Thanks @heffneil. I'm talking more about this integration working though and not needing to run something separate

**Comment by outdooricon** [comment]
Oops, accidentally closed

**Comment by heffneil** [comment]
Ok well it’s not a very challenging situation to setup the stand-alone and be out of the cloud requirements. Plus you get incredibly more information than the integration can provide. I don’t know your level of technical ability but we can help.

**Comment by ccutrer** [comment]
Yes, if you use https://github.com/ccutrer/waterfurnace_aurora/, you'll have to make a cable and buy (two) RS485 adapter(s), but it has no problems running multiple instances, integrates into Home Assistant just fine, gives you vastly more information and control, and isn't dependent on WaterFurnace's cloud services or the Aurora WebLink hardware (personally, mine has failed, and I don't even care).

**Comment by starsoccer** [comment]
+1 on getting support for multiple units or getting a nice mqtt wrapper integration for the ccutrer mqtt similar to tasmota where its auto discovered and imported.

If it helps Id gladly double the bounty and put in another 100$ to get this done. Ideally with controls if possible

**Comment by sam-kleiner** [comment]
@outdooricon if my PR https://github.com/sdague/waterfurnace/pull/11 gets merged donate the 100 to charity. After this change it should be trivial to add to hass.

The way I did it here I am imagining we would just configure multiple waterfurnace integrations, each with their own device selector in the yaml. I figured that was better than reworking this to pull from multiple devices simultaneously.
